<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>OpenWithDialog</class>
 <widget class="QDialog" name="OpenWithDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>311</width>
    <height>311</height>
   </rect>
  </property>
  <property name="focusPolicy">
   <enum>Qt::StrongFocus</enum>
  </property>
  <property name="windowTitle">
   <string>OpenWithDialog</string>
  </property>
  <widget class="QLabel" name="back">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>311</width>
     <height>311</height>
    </rect>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QFrame" name="panel">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>311</width>
     <height>311</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QWidget{
	background-color: rgba(255, 255, 255, 0);
}
QFrame {
	background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:0, y2:1, stop:0 rgba(46, 46, 44, 250), stop:1 rgba(97, 102, 113, 161));
   border-radius: 13px;
}
</string>
   </property>
   <layout class="QGridLayout" name="gridLayout_6">
    <item row="1" column="2">
     <widget class="QToolButton" name="btn6">
      <property name="minimumSize">
       <size>
        <width>90</width>
        <height>90</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>90</width>
        <height>90</height>
       </size>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>(6) Adicionar</string>
      </property>
      <property name="icon">
       <iconset resource="imgs/imgs.qrc">
        <normaloff>:/add.png</normaloff>:/add.png</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>64</width>
        <height>64</height>
       </size>
      </property>
      <property name="toolButtonStyle">
       <enum>Qt::ToolButtonTextUnderIcon</enum>
      </property>
      <property name="autoRaise">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="2" column="2">
     <widget class="QToolButton" name="btn9">
      <property name="minimumSize">
       <size>
        <width>90</width>
        <height>90</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>90</width>
        <height>90</height>
       </size>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>(9) Adicionar</string>
      </property>
      <property name="icon">
       <iconset resource="imgs/imgs.qrc">
        <normaloff>:/add.png</normaloff>:/add.png</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>64</width>
        <height>64</height>
       </size>
      </property>
      <property name="toolButtonStyle">
       <enum>Qt::ToolButtonTextUnderIcon</enum>
      </property>
      <property name="autoRaise">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QToolButton" name="btn1">
      <property name="minimumSize">
       <size>
        <width>90</width>
        <height>90</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>90</width>
        <height>90</height>
       </size>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>(1) Adicionar</string>
      </property>
      <property name="icon">
       <iconset resource="imgs/imgs.qrc">
        <normaloff>:/add.png</normaloff>:/add.png</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>64</width>
        <height>64</height>
       </size>
      </property>
      <property name="toolButtonStyle">
       <enum>Qt::ToolButtonTextUnderIcon</enum>
      </property>
      <property name="autoRaise">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QToolButton" name="btn7">
      <property name="minimumSize">
       <size>
        <width>90</width>
        <height>90</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>90</width>
        <height>90</height>
       </size>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>(7) Adicionar</string>
      </property>
      <property name="icon">
       <iconset resource="imgs/imgs.qrc">
        <normaloff>:/add.png</normaloff>:/add.png</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>64</width>
        <height>64</height>
       </size>
      </property>
      <property name="toolButtonStyle">
       <enum>Qt::ToolButtonTextUnderIcon</enum>
      </property>
      <property name="autoRaise">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="0" column="2">
     <widget class="QToolButton" name="btn3">
      <property name="minimumSize">
       <size>
        <width>90</width>
        <height>90</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>90</width>
        <height>90</height>
       </size>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>(3) Adicionar</string>
      </property>
      <property name="icon">
       <iconset resource="imgs/imgs.qrc">
        <normaloff>:/add.png</normaloff>:/add.png</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>64</width>
        <height>64</height>
       </size>
      </property>
      <property name="toolButtonStyle">
       <enum>Qt::ToolButtonTextUnderIcon</enum>
      </property>
      <property name="autoRaise">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QToolButton" name="btn2">
      <property name="minimumSize">
       <size>
        <width>90</width>
        <height>90</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>90</width>
        <height>90</height>
       </size>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>(2) Adicionar</string>
      </property>
      <property name="icon">
       <iconset resource="imgs/imgs.qrc">
        <normaloff>:/add.png</normaloff>:/add.png</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>64</width>
        <height>64</height>
       </size>
      </property>
      <property name="toolButtonStyle">
       <enum>Qt::ToolButtonTextUnderIcon</enum>
      </property>
      <property name="autoRaise">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QToolButton" name="btn8">
      <property name="minimumSize">
       <size>
        <width>90</width>
        <height>90</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>90</width>
        <height>90</height>
       </size>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>(8) Adicionar</string>
      </property>
      <property name="icon">
       <iconset resource="imgs/imgs.qrc">
        <normaloff>:/add.png</normaloff>:/add.png</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>64</width>
        <height>64</height>
       </size>
      </property>
      <property name="toolButtonStyle">
       <enum>Qt::ToolButtonTextUnderIcon</enum>
      </property>
      <property name="autoRaise">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QToolButton" name="btn5">
      <property name="minimumSize">
       <size>
        <width>90</width>
        <height>90</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>90</width>
        <height>90</height>
       </size>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>(5) Adicionar</string>
      </property>
      <property name="icon">
       <iconset resource="imgs/imgs.qrc">
        <normaloff>:/add.png</normaloff>:/add.png</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>64</width>
        <height>64</height>
       </size>
      </property>
      <property name="toolButtonStyle">
       <enum>Qt::ToolButtonTextUnderIcon</enum>
      </property>
      <property name="autoRaise">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QToolButton" name="btn4">
      <property name="minimumSize">
       <size>
        <width>90</width>
        <height>90</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>90</width>
        <height>90</height>
       </size>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>(4) Adicionar</string>
      </property>
      <property name="icon">
       <iconset resource="imgs/imgs.qrc">
        <normaloff>:/add.png</normaloff>:/add.png</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>64</width>
        <height>64</height>
       </size>
      </property>
      <property name="toolButtonStyle">
       <enum>Qt::ToolButtonTextUnderIcon</enum>
      </property>
      <property name="autoRaise">
       <bool>true</bool>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources>
  <include location="imgs/imgs.qrc"/>
 </resources>
 <connections/>
</ui>
